_data/meltano/utilities/postgres/postgres.yml

```diff
@@ -0,0 +1,41 @@
+name: postgres
+namespace: postgres
+domain_url: https://www.postgresql.org/
+label: Postgres
+description: The World's Most Advanced Open Source Relational Database.
+docs: https://www.postgresql.org/
+repo: https://github.com/postgres/postgres
+maintenance_status: active
+variant: postgres
+settings: []
+commands:
+  start:
+    args: start
+    container_spec:
+      image: postgres:15
+      env:
+        PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/lib/postgresql/15/bin
+        LANG: en_US.utf8
+        POSTGRES_DB: warehouse
```

Comment:
@aphethean1 definitely not blocking for this PR but it would be a nice follow on to use settings for the db/user/pass/etc. values so someone could override if they wanted to.